### PR TITLE
feat: initialize hapi<>proto converter module

### PR DESCRIPTION
**Pull request by epicadk**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

trivial

**Description**
Clear and concise code change description. 

initialize hapi<>proto converter module

[GSoC project](https://github.com/google/android-fhir/wiki/GSoC-Project-Ideas#hapi--proto-conversion-tool)

**Type**
Choose one:  Feature 

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

**Comment by jingtang10** [comment]
thanks for this @epicadk - is it possible for you to create a new branch, possibly `gsoc/hapi-proto-converter` or something similar and we can conduct reviews for PRs merging into that branch?

**Comment by epicadk** [comment]
> thanks for this @epicadk - is it possible for you to create a new branch, possibly `gsoc/hapi-proto-converter` or something similar and we can conduct reviews for PRs merging into that branch?

Sure.

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -26,6 +26,9 @@ object Dependencies {
     const val workRuntimeKtx = "androidx.work:work-runtime-ktx:${Versions.Androidx.workRuntimeKtx}"
   }
 
+  object Protobufs {
```
Comment (jingtang10) [inline buildSrc/src/main/kotlin/Dependencies.kt:29]:
```suggestion
  object FhirProto {
```

the project name is FHIR Proto - so I'd be tempted to follow that.

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 2]
```diff
@@ -106,7 +109,9 @@ object Dependencies {
       const val room = "2.3.0"
       const val workRuntimeKtx = "2.5.0"
     }
-
+    object Protobufs {
+      const val fhirprotobuf = "0.6.1"
+    }
```
Comment (jingtang10) [inline buildSrc/src/main/kotlin/Dependencies.kt:114]:
```suggestion
    object FhirProto {
      const val fhirProto = "0.6.1"
    }
```

**Review by jingtang10 — APPROVED** [review]

**Comment by jingtang10** [comment]
can you also please make sure your mentor gives a review before merging? thanks!

**Comment by epicadk** [comment]
> can you also please make sure your mentor gives a review before merging? thanks!

cc @Rkareko

**Review by Rkareko — APPROVED** [review]

**Comment by Rkareko** [comment]
@epicadk Could you have a look at the build failure

**Comment by epicadk** [comment]
> @epicadk Could you have a look at the build failure

The crash is due to failing tests in the reference app. You can view this is the log and I've also confirmed from the artifacts.

**Pull request merged**